### DISCUSSION TITLE
Lab 4, Exercise 3

### DISCUSSION THREAD

**Discussion by Iamnyatichi** [giswqs/geog-312#133] [Q&A] [answered]
I am getting the following error while loading an Interactive map in Google Colab. However, in VS Code, I can successfully load the map without any problems.
![image](https://github.com/user-attachments/assets/bdc1e932-a049-4586-b66c-5bf9ec245e09)

**Comment by giswqs** [comment, marked as answer]
```bash
pip install mapclassify
```
